ForgeUnity/Assets/BeardedManStudios/Scripts/Networking/Forge/Networking/Objects/NetworkObject.cs

```diff
@@ -883,10 +885,7 @@ private void ClearClientPendingRPC()
 				InvokeRpc(rpc.sender, rpc.timestep, rpc.data, rpc.receivers);
 
 			foreach (PendingLocalRPC rpc in pendingLocalRpcs)
-				if (rpc.Reliable)
-					SendRpc(rpc.TargetPlayer, rpc.MethodId, rpc.Args);
-				else
-					SendRpcUnreliable(rpc.TargetPlayer, rpc.MethodId, rpc.Args);
```

Comment:
Now that I see this red block I even question why we are adding any unreliable rpc calls into a list to send later, it kinda defeats the point of it being unreliable haha.